contracts/rust/deployer/README.md

```diff
@@ -567,3 +751,1002 @@ If the deployer can't find deployed contracts:
+
+```bash
+export OUTPUT_FILE=.env.eth.mainnet.lightclient.admin.transfer
+touch $OUTPUT_FILE
+```
+
+3. Set the environment variables for the admin transfer configuration
+
+```bash
+# Set the EspressoSys multisig address (current admin)
+export ESPRESSO_SEQUENCER_ETH_MULTISIG_ADDRESS=0xESPRESSOSYS_MULTISIG_ADDRESS
+
+# Set the OpsTimelock address (new admin)
+export ESPRESSO_SEQUENCER_OPS_TIMELOCK_ADDRESS=0xOPS_TIMELOCK_ADDRESS
+
+# Set the LightClientProxy address
+export ESPRESSO_SEQUENCER_LIGHT_CLIENT_PROXY_ADDRESS=0xLIGHT_CLIENT_PROXY_ADDRESS
+```
+
+4. Run the docker-compose command to create the multisig proposal for admin transfer
+
+```bash
+docker compose run --rm \
+  -e RPC_URL \
+  -e ESPRESSO_SEQUENCER_ETH_MNEMONIC \
+  -e ESPRESSO_SEQUENCER_ETH_MULTISIG_ADDRESS \
+  -e ESPRESSO_SEQUENCER_OPS_TIMELOCK_ADDRESS \
+  -e ESPRESSO_SEQUENCER_LIGHT_CLIENT_PROXY_ADDRESS \
+  -v $(pwd)/$OUTPUT_FILE:/app/$OUTPUT_FILE \
+  \
+  deploy-sequencer-contracts \
+  deploy --propose-transfer-ownership-to-timelock \
+  --target-contract lightclient \
+  --timelock-address $ESPRESSO_SEQUENCER_OPS_TIMELOCK_ADDRESS \
+  --rpc-url=$RPC_URL \
+  --out $OUTPUT_FILE
+  # if doing a real run then add --dry-run
+```
+
+**Note**: The admin transfer process will:
+
+- Create a multisig proposal to call `transferOwnership(address)` on the LightClientProxy
+- Set the OpsTimelock as the new admin/owner of the LightClientProxy
+- Maintain the proxy's functionality while changing administrative control
+
+5. Verify the admin transfer proposal was created
+
+You should see output similar to:
+`LightClientProxy ownership transfer proposal sent. Send this link to the signers to sign the proposal: https://app.safe.global/transactions/queue?safe=0xESPRESSOSYS_MULTISIG_ADDRESS`
+
+6. After the signer threshold signs the proposal and one executes the proposal, verify the admin transfer on-chain
+   (assuming you have Foundry installed)
+
+```bash
+# First, source the output file to load the deployed contract addresses
+source $OUTPUT_FILE
+
+# Verify the variables are loaded correctly
+echo "OpsTimelock Address: $ESPRESSO_SEQUENCER_OPS_TIMELOCK_ADDRESS"
+
+# Verify the OpsTimelock is now the owner
+cast call $ESPRESSO_SEQUENCER_LIGHT_CLIENT_PROXY_ADDRESS "owner()(address)" --rpc-url $RPC_URL | grep -i $ESPRESSO_SEQUENCER_OPS_TIMELOCK_ADDRESS
+```
+
+#### Step 5: Deploy EspToken with SafeExitTimelock as Admin
+
+```bash
+export OUTPUT_FILE=.env.eth.mainnet.esptoken
+touch $OUTPUT_FILE
+```
+
+3. Set the environment variables for the EspToken deployment configuration
+
+```bash
+# Set the SafeExitTimelock address (will be the admin of EspTokenProxy)
+export ESPRESSO_SEQUENCER_SAFE_EXIT_TIMELOCK_ADDRESS=0xSAFE_EXIT_TIMELOCK_ADDRESS
+
+# Set the initial token supply (optional, will use default if not set)
+export ESPRESSO_SEQUENCER_ESP_TOKEN_INITIAL_SUPPLY=3590000000000000000000000000 # 3.59 billion tokens with 18 decimals
```

Comment:
I think this env var doesn't have the ESPRESSO_SEQUENCER_ prefix

https://github.com/EspressoSystems/espresso-sequencer/blob/d805a1af47645b05ca187bb6c9bdbfdbeb871ac5/sequencer/src/bin/deploy.rs#L237

Reply:
And I think we removed the default.